# msg/RestrictionArray.msg
Header header
Restriction[] restrictions  # 禁区数组